packages/kotti-ui/source/kotti-filters/components/FilterActions.vue

```diff
@@ -1,12 +1,14 @@
 <template>
 	<div class="kt-filter__actions">
 		<ButtonLink
+			:data-test="dataTest ? `${dataTest}-add-filter` : null"
 			:isDisabled="isAddDisabled"
 			:isLoading="isLoading"
 			:label="translations.addFilterLabel"
 			@click="handleAdd"
 		/>
 		<ButtonLink
+			:data-test="dataTest ? `${dataTest}-clear-all` : null"
```

Comment:
Should either be `:dataTest=` or `: undefined` depending on if this is intended to be passed as a prop or as a html attribute